nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/Gather.java

```diff
@@ -135,7 +156,7 @@ public String opName() {
         //Use scatter add plus permute
         SDVariable dimsExAxis = sameDiff.range(null, sameDiff.constant(0), rank, sameDiff.constant(1), DataType.INT);
         SDVariable axisRank1 = axis.reshape(1);
-        dimsExAxis = sameDiff.math().listDiff(dimsExAxis, axisRank1)[0];  //Don't need indices
+        dimsExAxis = sameDiff.f().listdiff(dimsExAxis, axisRank1)[0];  //Don't need indices
```

Comment:
samediff.f() (i.e., DifferentialFunctionFactory) is going away soon.
Is this method not available in any namespace? If so, we should fix that.